saleor/product/tasks.py

```diff
@@ -43,8 +42,11 @@ def update_variants_names(product_type_pk: int, saved_attributes_ids: List[int])
 
 @app.task
 def update_product_minimal_variant_price_task(product_pk: int):
-    product = Product.objects.get(pk=product_pk)
-    update_product_minimal_variant_price(product)
+    # TODO: We should refactor this in separete PR
+    # https://app.clickup.com/t/6a5txz
+    return
+    # product = Product.objects.get(pk=product_pk)
```

Comment:
This comment is necessary?

Reply:
This code should be refactored in next PR's I prefer left it commented instead of removing it. It is easier to look at how this code looks before my changes if the old code is commented instead of looking for it in repository/history.